server/constants/tours/eyeSpy/june-2022.json

```diff
@@ -0,0 +1,95 @@
+{
+  "title": "Eye Spy Gallery Scavenger Hunt",
+  "subtitle": "",
+  "heroImageInvno": "BF961",
+  "heroImageStyle": {
+    "overlay": {
+      "backgroundColor": "rgba(0, 0, 0, 0.4)"
+    }
+  },
+  "description": "<p>Can you spy all of these works in our galleries? Check them off when you find them and turn in the completed sheet for a prize! </p>",
+  "body": {
+    "from": 0,
+    "size": 25,
+    "_source": [
+      "id",
+      "title",
+      "people",
+      "medium",
+      "imageOriginalSecret",
+      "imageSecret",
+      "ensembleIndex",
+      "objRightsTypeId",
+      "onview",
+      "invno",
+      "image",
+      "curatorialApproval",
+      "shortDescription",
+      "nationality",
+      "birthDate",
+      "deathDate",
+      "artistPrefix",
+      "artistSuffix",
+      "culture",
+      "displayDate",
+      "medium",
+      "dimensions",
+      "creditLine",
+      "longDescription",
+      "bibliography",
+      "exhHistory",
+      "publishedProvenance"
+    ],
+    "query": {
+      "bool": {
+        "filter": {
+          "terms": {
+            "invno": [
+              "BF819",
+              "BF961",
+              "BF990",
+              "BF2553",
+              "BF335",
+              "BF854",
+              "BF2075",
+              "BF888",
+              "BF582",
+              "BF350"
```

Comment:
Can you change all of these tow lower case (eg: bf819)? The invno needs to be lowercase for the query to work!